MapboxCoreNavigation/MGLMapboxEvents.swift

```diff
@@ -0,0 +1,39 @@
+import Mapbox
+import Polyline
+
+extension MGLMapboxEvents {
+    class func addDefaultEvents(routeProgress: RouteProgress, sessionIdentifier: UUID) -> [String: Any] {
+        var modifiedEventDictionary: [String: Any] = [:]
+    
+        modifiedEventDictionary["platform"] = UIDevice.current.systemName
+        modifiedEventDictionary["operatingSystemVersion"] = "\(UIDevice.current.systemName)-\(UIDevice.current.systemVersion)"
+        modifiedEventDictionary["sdkIdentifier"] = "mapbox-navigation-ios"
+        modifiedEventDictionary["sdkVersion"] = "0.4.0"
```

Comment:
Something like: `Bundle.navigationUI.object(forInfoDictionaryKey: "CFBundleShortVersionString")`. There should be a constant for CFBundleShort… IIRC